package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aserto/aserto-react-components",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
stricter types means some chance that dependents need to update their code when upgrading